content/overview/pricing/rates.md

```diff
@@ -33,31 +33,31 @@ You can email [cloud-gov-inquiries@gsa.gov](mailto:cloud-gov-inquiries@gsa.gov)
 The [prototyping package]({{< relref "overview/pricing/prototyping.md" >}}) lets many teams build and deploy apps, with full access control. Prototyping packages cannot host production data; all apps are limited to the `*.app.cloud.gov` domain. Whole agencies or departments often purchase a single prototyping package.
 
 **Annual fee:** $15K<br>
-**Resource usage quota:** ~$99 per GB per month[*](#quota-pricing), up to any limit set by user<br>
+**Resource usage quota:** ~$100 per GB per month[*](#quota-pricing), up to any limit set by user<br>
```

Comment:
Our price for usage is always the same regardless of package. Do we need to repeat it in every description? Is there a way to make this more compact? 

The limits also do not change, except for Open Data.